File: articles/algorithms-in-decision-science/java/ExpectedValue.java
public class ExpectedValue {
    public static void main(String[] args) {
        double probability = 0.82;
        double benefitIfAct = 0.88;
        double costIfAct = 0.30;
        double expectedValue = probability * benefitIfAct - costIfAct;
        System.out.printf("expected_value_of_action=%.4f%n", expectedValue);
    }
}

File: articles/algorithms-in-decision-science/go/expected_value.go
package main

import "fmt"

func main() {
	probability := 0.82
	benefitIfAct := 0.88
	costIfAct := 0.30
	expectedValue := probability*benefitIfAct - costIfAct
	fmt.Printf("expected_value_of_action=%.4f\n", expectedValue)
}
